readalongs/cli.py

```diff
@@ -66,116 +73,132 @@ def cli():
+            text_language=kwargs["language"],
+            save_temps=(output_base if kwargs["save_temps"] else None),
+        )
+    if kwargs["output_xhtml"]:
+        tokenized_xml_path = "%s.xhtml" % output_base
     else:
-        _, input_ext = os.path.splitext(kwargs['inputfile'])
-        tokenized_xml_path = '%s%s' % (output_base, input_ext)
-    if os.path.exists(tokenized_xml_path) and not kwargs['force_overwrite']:
-        raise click.BadParameter("Output file %s exists already, did you mean to do that?"
-                                 % tokenized_xml_path)
-    smil_path = output_base + '.smil'
-    if os.path.exists(smil_path) and not kwargs['force_overwrite']:
-        raise click.BadParameter("Output file %s exists already, did you mean to do that?"
-                                 % smil_path)
-    _, wav_ext = os.path.splitext(kwargs['wavfile'])
-    wav_path = output_base + wav_ext
-    if os.path.exists(wav_path) and not kwargs['force_overwrite']:
-        raise click.BadParameter("Output file %s exists already, did you mean to do that?"
-                                 % wav_path)
+        _, input_ext = os.path.splitext(kwargs["inputfile"])
+        tokenized_xml_path = "%s%s" % (output_base, input_ext)
+    if os.path.exists(tokenized_xml_path) and not kwargs["force_overwrite"]:
+        raise click.BadParameter(
+            "Output file %s exists already, did you mean to do that?"
+            % tokenized_xml_path
+        )
+    smil_path = output_base + ".smil"
+    if os.path.exists(smil_path) and not kwargs["force_overwrite"]:
+        raise click.BadParameter(
+            "Output file %s exists already, did you mean to do that?" % smil_path
+        )
+    _, audio_ext = os.path.splitext(kwargs["audiofile"])
+    audio_path = output_base + audio_ext
+    if os.path.exists(audio_path) and not kwargs["force_overwrite"]:
+        raise click.BadParameter(
+            "Output file %s exists already, did you mean to do that?" % audio_path
+        )
     unit = kwargs.get("unit", "w")
     bare = kwargs.get("bare", False)
-    if not unit:     # .get() above should handle this but apparently the way kwargs is implemented
-        unit = "w"   # unit could still be None here.
+    if (
+        not unit
+    ):  # .get() above should handle this but apparently the way kwargs is implemented
+        unit = "w"  # unit could still be None here.
     try:
-        results = align_audio(kwargs['inputfile'], kwargs['wavfile'],
-                              unit=unit,
-                              bare=bare,
-                              save_temps=(output_base
-                                          if kwargs['save_temps'] else None))
+        results = align_audio(
+            kwargs["inputfile"],
+            kwargs["audiofile"],
+            unit=unit,
+            bare=bare,
+            config=config,
+            save_temps=(output_base if kwargs["save_temps"] else None),
+        )
```

Comment:
This does improve legibility, so I agree with the extra lines here, when there is a good legibility benefit.